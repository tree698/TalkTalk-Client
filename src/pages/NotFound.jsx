import React from 'react';

export default function NotFound() {
  return <h1>Page is not found! 😝</h1>;
}
