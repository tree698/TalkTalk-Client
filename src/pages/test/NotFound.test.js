import { Route } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { withRouter } from '../../test/utils';
import NotFound from '../NotFound';

describe('Not Found', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      withRouter(<Route path="/" element={<NotFound />} />)
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
