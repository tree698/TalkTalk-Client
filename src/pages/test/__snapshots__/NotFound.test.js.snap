// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found renders correctly 1`] = `
<h1>
  Page is not found! 😝
</h1>
`;
